clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnection.java

```diff
@@ -68,6 +68,10 @@ public HubConnection(String url, Transport transport, Logger logger){
                     case STREAM_INVOCATION:
                     case STREAM_ITEM:
                     case CLOSE:
```

Comment:
As written the Close logic runs for both  `STREAM_INVOCATION` and `STREAM_ITEM` message types. I need to move those two cases with the other unsupported message types.

Reply:
And a note to myself to add tests that verify exceptions are thrown when unsupported message types are received. Those kind of tests would have caught this